backend/molgenis-emx2-graphql/src/test/java/org/molgenis/emx2/graphql/TestGraphqlAdminFields.java

```diff
@@ -57,6 +60,83 @@ public void testUsers() {
         });
   }
 
+  @Test
+  void testUpdateUser() {
+    database.tx(
+        testDatabase -> {
```

Comment:
why the test also in a tx?

Reply:
I looked at the test above as an example, which does this. It also made sense to me, since if the test fails, I'd like it not to leave any artifacts that might break other things.